docs/developing.rst

```diff
@@ -119,8 +123,79 @@ Start by following the steps here: https://github.com/edx/edx-proctoring
         * Go to ``/admin/verify_student/manualverification/`` on your sandbox
         * Create a record for the given user, with status "approved".
 
+Backend-specific Information
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+One of the main motivations for setting up a sandbox to test
+proctoring is having an externally accessible system which can be
+accessed by our proctoring providers' systems. This enables more
+thorough end-to-end testing.
+
+To enable proctoring in a way that won't be overridden by ansible
+plays, you can add the following to a sandbox's
+``/edx/app/edx_ansible/server-vars.yml`` at the end of the
+``EDXAPP_FEATURES`` array::
+
+  EDX_APP_FEATURES:
+    MILESTONES_APP: true
+    ...
+    ENABLE_API_DOCS: true
+    ENABLE_SPECIAL_EXAMS: true
+
+  PROCTORING_SETTINGS:
+    MUST_BE_VERIFIED_TRACK: False
+
+  EDXAPP_PROCTORING_BACKENDS:
+    ...
+
+Placing these configurations here (rather than the more generic
+locations mentioned in `the README`_) will allow us to leverage the
+power of the ansible playbooks used to construct and administer
+sandboxes.
+
+The contents of ``EDXAPP_PROCTORING_BACKENDS`` will depend on which
+backend you're interested in testing. It's nececssary to provide a
```

Comment:
```suggestion
backends you're interested in testing. It's necessary to provide a
```
Re: `backend` versus `backends`, the way this is phrased implies, to me, that the sandbox can be configured with only one backend. I think making backends plural clarifies that you can configure your sandbox with more than one backend, which may be a helpful distinction for people new to proctoring.